journal-week-1.md

```diff
@@ -15,4 +15,6 @@
  
  <img src="http://media.giphy.com/media/DdQaV2l2ltSjC/giphy.gif"</>
  
+ (Also, I won't give up because I need a job in order to get a cat.)
+ 
```

Comment:
Fun essay! Thanks for your reflection. I'd love to see you tackle a big concept (like pair programming).

(I also became a dev to get a cat, and now I have two, and they _run my house_.)